tests/test_fxdc_core.py

```
@@ -1,49 +1,55 @@
+
 
 def test_basic_primitives_roundtrip():
     example = 'name|str = "John"\nage|int = 23'
     obj = loads(example)
-    data = obj.to_dict() if hasattr(obj, 'to_dict') else obj.original
+    data = obj.to_dict() if hasattr(obj, "to_dict") else obj.original
     assert data == {"name": "John", "age": 23}
 
     round = dumps(data)
-    assert 'name|str' in round and 'age|int' in round
+    assert "name|str" in round and "age|int" in round
```

Ruff (0.12.2)

Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)